src/Neo/SmartContract/Native/NeoToken.cs

```diff
@@ -226,6 +227,7 @@ internal override ContractTask Initialize(ApplicationEngine engine)
             engine.Snapshot.Add(CreateStorageKey(Prefix_Committee), new StorageItem(cachedCommittee));
             engine.Snapshot.Add(CreateStorageKey(Prefix_VotersCount), new StorageItem(System.Array.Empty<byte>()));
             engine.Snapshot.Add(CreateStorageKey(Prefix_GasPerBlock).AddBigEndian(0u), new StorageItem(5 * GAS.Factor));
+            engine.Snapshot.Add(CreateStorageKey(Prefix_GasPerBlockCurrent), new StorageItem(new GasPerBlock() { Gas = 5 * GAS.Factor, PreviousGas = 5 * GAS.Factor, Index = 0 }));
```

Comment:
```suggestion
            engine.Snapshot.Add(CreateStorageKey(Prefix_GasPerBlockCurrent).AddBigEndian(0u), new StorageItem(5 * GAS.Factor));
```

This way you can use `backwards` for lookup on the index

Reply:
This pr is for avoid `backwards`